src/emailcontact.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020 Open Mobile Platform LLC.
+ *
+ * This program is licensed under the terms and conditions of the
+ * Apache License, version 2.0.  The full text of the Apache License is at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#ifndef EMAILCONTACT_H
+#define EMAILCONTACT_H
+
+#include <QMailAddress>
+#include <QMailTimeStamp>
+
+
+class Q_DECL_EXPORT EmailContact
```

Comment:
Probably not necessary, but could potentially make this a Q_GADGET if we ever think it might be useful to expose the type directly to QML (rather than field-wise via the model roles).  But, of course, we can do that when needed, rather than doing so speculatively now, so, never mind.

Reply:
At the moment, in QML this class is not used directly, only as part the model.  Now you don't need to change it to Q_GADGET.  Maybe later